lf-to-abstract-sql.ometajs

```diff
@@ -1125,8 +1124,8 @@ LF2AbstractSQL.CreateLinkTableResolver = function(query, linkTableAlias, bind, r
 		);
 	};
 	// If it's queued run it immediately, if it's already been done then that means this is a different link table from the same resource
-	// and we should also immediately resolve for this one
-	if (this.linkTableResolvers[parentAlias] === LINK_RESOLVE_QUEUED || this.linkTableResolvers[parentAlias] === LINK_RESOLVE_DONE) {
+	// and we should also immediately resolve for this one, and if the bind is for a numbered identifier then assume it will be used at some
```

Comment:
```suggestion
	// and we should also immediately resolve for this one, and if the bind is for a numbered identifier then assume it will be used at some point
```